cmd/eskip/writeclient.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import "github.com/zalando/skipper/eskip"
+import (
+	"errors"
+	etcdclient "github.com/zalando/skipper/etcd"
```

Comment:
yep, sorry i mean as 'not very useful'. I see now, it is because of the constants `etcd` and `innkeeper`. (y)